Makefile

```diff
@@ -20,9 +23,6 @@ ptx:
 spirv:
 	bin/compile --jdk jdk21 --backends spirv,ptx,opencl
 
-offline:
```

Comment:
why we removed the offline build?

Reply:
This rule is not used now. Even without connection, and assuming all dependencies are already installed, we can still build.

Reply:
thanks
should we clean it from the compile script as well?

Reply:
Yes, good catch.